packages/web/src/client/internal/open-feature-client.ts

```diff
@@ -181,6 +182,21 @@ export class OpenFeatureClient implements Client {
     return this.evaluate<T>(flagKey, this._provider.resolveObjectEvaluation, defaultValue, 'object', options);
   }
 
+  track(occurrenceKey: string, occurrenceDetails: TrackingEventDetails): void {
+
+    this.shortCircuitIfNotReady();
```

Comment:
Do we want to throw if the provider is unhealthy? I expected the track method to never throw.

Reply:
We want to throw here but catch it, I think. I just made it short circuit like all the other methods.

Reply:
done